lib/icinga/checkable-flapping.cpp

```diff
@@ -64,10 +64,16 @@ void Checkable::UpdateFlappingStatus(bool stateChange)
 	SetFlappingBuffer(stateChangeBuf.GetValue());
 	SetFlappingIndex(oldestIndex);
 	SetFlappingCurrent(flappingValue);
-	SetFlapping(flapping, true);
 
-	if (flapping != GetFlapping())
-		SetFlappingLastChange(Utility::GetTime());
+	if (flapping != GetFlapping()) {
+		SetFlapping(flapping, true);
```

Comment:
Logic change, why?

Reply:
We listen for `SetFlapping()` and want to be only notified when it actually changes.